pkgs/servers/honk/default.nix

```diff
@@ -1,21 +1,32 @@
-{ lib, buildGoModule, fetchurl, installShellFiles, sqlite }:
+{ lib
+, buildGoModule
+, fetchurl
+, sqlite
+, installShellFiles
+}:
 
 buildGoModule rec {
   pname = "honk";
-  version = "0.9.91";
+  version = "1.0.0";
 
   src = fetchurl {
     url = "https://humungus.tedunangst.com/r/honk/d/honk-${version}.tgz";
-    hash = "sha256-+NFWTTMVdngWsC8/EIN2xJC/5C4naaAekk/YoA17wFk=";
+    hash = "sha256-+0W9HncN+51dRE9bWJU4cAfYOc5bxNAqPe4xY+4UFg0=";
   };
   vendorHash = null;
 
-  buildInputs = [ sqlite ];
-  nativeBuildInputs = [ installShellFiles ];
+  buildInputs = [
+    sqlite
+  ];
+
+  nativeBuildInputs = [
+    installShellFiles
+  ];
+
   subPackages = [ "." ];
 
   postPatch = ''
-    substituteInPlace honk.go --replace \
+    substituteInPlace main.go --replace \
       "var viewDir = \".\"" \
```

Comment:
Ideally upstream would have supported changing this variable via an ldFlag or a command line argument...

Reply:
That's actually possible to add the `-viewdir` parameter to honk. This is just to have something working out of the box without adding any extra argument.

Reply:
> That's actually possible to add the `-viewdir` parameter to honk. This is just to have something working out of the box without adding any extra argument.

I see. I saw your [honk module PR](https://github.com/NixOS/nixpkgs/pull/249765) and I won't be able to review it. But I think it's OK to keep this substitution as is, but perhaps it'd be ideal to maybe remove it in favor of the module taking care of it. I am just thinking out loud, no action is strictly requested :).

Reply:
Of course the option of having this default parameter set in `ldFlags` is even better.

Reply:
I agree. Is it ok for you if I merge this?

Reply:
Maybe it'd be nice to add a comment near that hook that ideally the situation could be better, or that this is just for setting the default value of this parameter, something like that.

Reply:
I will immediately do that.

Reply:
Done, is it ok?

Reply:
Yes good enough :) LGTM now, when CI is green.